stdlib/2and3/filecmp.pyi

```diff
@@ -0,0 +1,49 @@
+# Stubs for filecmp (Python 2/3)
+import sys
+from typing import AnyStr, Callable, Dict, Generic, Iterable, List, Optional, Sequence, Tuple
+
+_SupportsBool = object
+
+DEFAULT_IGNORES = ...  # type: List[str]
+
+def cmp(f1: AnyStr, f2: AnyStr, shallow: _SupportsBool = ...) -> bool: ...
```

Comment:
I'm not sure AnyStr is correct here. Jukka has some draft guidance in https://github.com/python/typeshed/issues/871, which says that AnyStr should be used only when there are multiple arguments that must be of the same type. I don't think that's the case here.